### PR TITLE
Reverse proxy in the server

### PR DESCRIPTION
**Pull request by jkminneti**
ty quinn

### CODE REVIEW

File: pkg/adminserver/server.go [hunk 1]
```diff
@@ -142,30 +126,32 @@ func (s server) receiveAuthRedirect(w http.ResponseWriter, r *http.Request) {
...
+	remote, err := url.Parse(proxyUrl)
+	if err != nil {
+		return err
+	}
+
+	proxy := httputil.NewSingleHostReverseProxy(remote)
+
 	mux := http.NewServeMux()
+	mux.HandleFunc("/tuber/", func(w http.ResponseWriter, r *http.Request) { s.requireAuth(proxy).ServeHTTP(w, r) })
+	mux.HandleFunc("/tuber/_next/", func(w http.ResponseWriter, r *http.Request) { proxy.ServeHTTP(w, r) })
```
Comment (jkminneti) [inline pkg/adminserver/server.go:150]: the meat of it

File: cmd/env.go [hunk 2]
```diff
@@ -244,29 +242,27 @@ func getAllEnvGraphqlQuery(appName string) (map[string]string, error) {
...
 		}
 	`
 
-	var respData []*model.Tuple
+	var respData struct {
+		GetAppEnv []*model.Tuple
+	}
 
-	err = graphql.Query(context.Background(), gql, &respData, graph.WithVar("name", appName))
+	err = graphql.Query(context.Background(), fmt.Sprintf(gql, appName), &respData)
```
Comment (Hbbb) [inline cmd/env.go:257]: Wait did all this stuff not work before? This seems unrelated to the reverse proxy changes.

**Review by Hbbb — APPROVED** [review]
This is so sweet!

File: cmd/env.go (hunk 2 above)
Comment (jkminneti) [reply cmd/env.go:257]: correct. yeah should probably have separated it out but idk meh
